org.aposin.licensescout.licensereport/launch/org.aposin.licensescout.licensereport_scan.launch

```diff
@@ -1,19 +1,20 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<launchConfiguration type="org.eclipse.m2e.Maven2LaunchConfigurationType">
-    <booleanAttribute key="M2_DEBUG_OUTPUT" value="false"/>

```

Comment:
This indentation changes hide the real modification - could you revert where there is no change in the line?

Reply:
The file had the wrong line endings (Windows). I fixed it to Unix with this change.
We really should make sure that all files in the repository have unique line endings.

Reply:
Looks like the problem is still here could you please double check?